common-ui/common-ui-devstart.rb

```diff
@@ -36,7 +36,7 @@ def swagger_regen(cmd_name)
   common.run_inline %W{yarn run codegen}
 end
 
-def build(cmd_name, args)
+def build(cmd_name, ui_name, args)
```

Comment:
not provided by callers?

Reply:
what does that mean?

Reply:
Nevermind - I thought this PR introduced `@ui_name`